mono/mini/exceptions-amd64.c

```diff
@@ -1122,6 +1128,14 @@ init_table_no_lock (void)
 			g_rtl_delete_growable_function_table = (RtlDeleteGrowableFunctionTablePtr)GetProcAddress (g_ntdll, "RtlDeleteGrowableFunctionTable");
 		}
 
+		// Fallback on systems not having RtlAddGrowableFunctionTable.
+		if (g_rtl_add_growable_function_table == NULL) {
+			if (GetModuleHandleEx (0, TEXT ("kernel32.dll"), &g_kernel32dll) == TRUE) {
```

Comment:
Comparison to TRUE is a mistake. The success values are non-zero, not specifically  TRUE. Just say `if (GetMod..())` not `if (GetMod...() == TRUE)`

Is anything really ever in kernel32.dll? Is anything ever missing from ntdll.dll? I think you can just use ntdll.dll.

Reply:
Also, this TEXT business is a Win9x compat layer. Nobody needs that any  longer.
Say `GetModuleHandleExW(0, L"kernel32.dll")...`

Reply:
Ok, well MSDN, does say kernel32.dll. Still probably just a forwarder to ntdll and just use ntdll. "Rtl" basically means "ntdll".

Reply:
Another option to strongly consider:
 The v1 (XP) functions are present on all systems, in ntdll.lib.
 Just build a custom ntdll.lib to bind to them directly -- no LoadLibrary/GetProcAddress then.

Reply:
Will fix the compare against TRUE to none zero.